index.html

```diff
@@ -540,8 +540,8 @@ <h4>F-Droid</h4>
 
                                             </div>
                                             <div class="item">
-                                                <p>F-Droid also guarantees the best possible privacy and therefore does not track its users. <br />
-                                                    At this point we would like to thank F-Droid for providing the downloads.
+                                                <p>F-Droid also guarantees the best possible privacy and does not track its users.<br />
+                                                    At this point, we would like to thank F-Droid for providing their services.
```

Comment:
We should probably point out what services we use from them. Right now, it's pretty much building and distributing NewPipe APKs safely and reproducibly.

Also, you don't really "provide" downloads, you "host" them. But that's just my personal preference, I guess. We should reword the entire sentence.

Reply:
lets do it then.

Maybe we want to link to their terms in that sentence? https://f-droid.org/en/about/#terms-etc

_F-Droid respects your privacy. They don’t track you, or your device. We would like to thank them for providing their services in this way._

Reply:
Could I get feedback to that guys?

Reply:
> F-Droid respects your privacy. They don’t track you, or your device. We would like to thank them for providing their services in this way.

sounds good to me.

Reply:
done